public_html/wp-content/themes/pattern-directory/functions.php

```diff
@@ -264,6 +264,10 @@ function rewrite_search_url() {
 		wp_redirect( home_url( '/search/' ) . urlencode( trim( get_query_var( 's' ) ) ) . '/' );
 		exit();
 	}
+	if ( preg_match( '/^my-favorites(.*)/', trim( $_SERVER['REQUEST_URI'], '/' ), $matches ) ) {
```

Comment:
Is it worth leaving a note here?... assuming that for a future developer this is a bit unexpected.